keps/sig-storage/2317-fsgroup-on-mount/README.md

```diff
@@ -306,12 +324,8 @@ the existing API objects?**
 operations covered by [existing SLIs/SLOs]?**
   Think about adding additional work or introducing new steps in between
   (e.g. need to do X to start a container), etc. Please describe the details.
-  
-  Depending on the driver implementation of applying FSGroup, latency for the following SLI may increase:
-  
-  "Startup latency of schedulable stateful pods, excluding time to pull images, run init containers, provision volumes (in delayed binding mode) and unmount/detach volumes (from previous pod if needed), measured from pod creation timestamp to when all its containers are reported as started and observed via watch, measured as 99th percentile over last 5 minutes"
-  
-  Comparing to the existing recursive `chown` and `chmod` strategy, this operation will likely improve pod startup latency in the most common case.
+
+  This feature does not have any impact on scalability.
 
```

Comment:
I agree with this change. This feature fixes a "correctness" issue  but cost of  the fix is basically an extra attribute in already existing CSI RPC call (NodeStage or NodePublish) and should not result in any change in SLIs/SLOs.